CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing Guide
+
+Thanks for your interest in contributing! All of our lessons are a "Work in Progress" and we welcome new people willing to help out.
+
+## Consider the scope
+
+To help us out, please consider the scope of your change(s). If the change is a typo or other minor change, please [make a Pull Request](#making-a-pull-request).
+
+If you would like to change significant parts of a lesson, please first [discuss it with us](#discussing-your-change). The section of content you wish to change might be in need of significant work and we would happy accept a contribution. Other sections may be more settled and require more discussion.
+
+## Discussing your change
+
+When making a significant change we ask that you discuss it with us first. This is because we do have previous experience teaching our students with the existing content. It is important for us to retain this because we have learnt which strategies tend to work better than others.
+
+Please create an issue describing your change and the reasoning behind it.
```

Comment:
I'd like to also suggest the module Slack channel. But I'm not sure we have a public one for this module. Unless #programming-fundamentals is intended to do that?

Reply:
One to think about in the future.